Version: validate_mapping_file.py 1.8.0
